lib/galaxy/webapps/galaxy/controllers/requests_common.py

```diff
@@ -163,7 +164,7 @@ def dataset_transfer_status_updates( self, trans, ids=None, transfer_status_list
                     sample_dataset = trans.sa_session.query( self.app.model.SampleDataset ).get( trans.security.decode_id( id ) )
                     if sample_dataset.status != transfer_status:
                         rval[ id ] = { "status": sample_dataset.status,
-                                       "html_status": unicode( trans.fill_template( "requests/common/sample_dataset_transfer_status.mako",
+                                       "html_status": text_type( trans.fill_template( "requests/common/sample_dataset_transfer_status.mako",
```

Comment:
@nsoranzo
I'm not sure this one ... this might actually break on python 3 if the response is `str`?